go-controller/pkg/cni/utils.go

```diff
@@ -117,13 +118,17 @@ func PodAnnotation2PodInfo(podAnnotation map[string]string, podNADAnnotation *ut
 		return nil, err
 	}
 
+	// For kubevirt live migratable interface the ip config is done with dhcp
+	skipIPConfig := netName == types.DefaultNetworkName && kubevirt.AllowPodBridgeNetworkLiveMigration(podAnnotation)
```

Comment:
nit: could you export this into a properly named func ? Something like `func isKubeVirtLiveMigratableInterface(podAnnotation)`.

That would make the comment redundant.

Reply:
Well how we discover the network info between virt-launcher pods is quite kubevirt centric, I am not sure we are going to be able to generalize this, maybe some parts of it, will try my best